### ISSUE TITLE
Add carbon emissions to model training

### ISSUE THREAD

**Issue by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#22]
Add the carbon emissions to the model training script. For example using [codecarbon](https://github.com/mlco2/codecarbon/).

**Comment by KennethEnevoldsen** [comment]
@rlrs This is an old issue, but does the existing implementation log anything regarding power consumption etc.?

**Comment by rlrs** [comment]
Yes, energy usage is logged in wandb for example. However, LUMI is carbon neutral afaik.

**Comment by KennethEnevoldsen** [comment]
Yes, it runs on clean energy (so carbon neutral at least in the running costs). I am not sure if that is realized 100 green or more like in practice it sometimes uses non-green sources depending on the grid, however, that is not out problem (at least not atm.).

Perfect. I will close this issue for now then.
